homeassistant/components/reolink/select.py

```diff
@@ -175,6 +175,62 @@ def _get_quick_reply_id(api: Host, ch: int, mess: str) -> int:
         value=lambda api, ch: HDREnum(api.HDR_state(ch)).name,
         method=lambda api, ch, name: api.set_HDR(ch, HDREnum[name].value),
     ),
+    ReolinkSelectEntityDescription(
+        key="main_frame_rate",
+        cmd_key="GetEnc",
+        translation_key="main_frame_rate",
+        entity_category=EntityCategory.CONFIG,
+        entity_registry_enabled_default=False,
+        get_options=lambda api, ch: [
+            f"{v} Hz" for v in api.frame_rate_list(ch, "main")
```

Comment:
Why are you adding `Hz` everywhere instead of using the unit of measurement?

Reply:
Because its a select entity which does not have a unit of measurement

Reply:
I asked on discord and select is the most appropriate because only discrete irregular integer values are allowed.

My case: frame rate: [4, 7, 10, 15] Hz 
Or for a different cam: [2, 4, 6, 8, 10, 12, 15, 16, 18, 20]
(Note the 12, 15, 16 is not equal step)

Reply:
Oh wait I am stupid, there is a unit of measurement for select!
Will change this

Reply:
Done, thanks for the feedback.
Much cleaner now!